src/Controller/Async/Stack.php

```diff
@@ -85,4 +127,22 @@ public function show(Request $request)
 
         return $this->render($template, ['context' => $context]);
     }
+
+    /**
+     * Perform, and return, a sub request for a rendered HTML stack item.
+     *
+     * @param Request $request
+     * @param string  $routeName
+     * @param string  $fileName
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    private function makeItemSubRequest(Request $request, $routeName, $fileName)
+    {
+        $panelUri = $this->generateUrl($routeName, ['fileName' => urlencode($fileName)]);
+        $subRequest = Request::create($panelUri, Request::METHOD_GET, [], $request->cookies->all(), [], $request->server->all());
+        $subRequest->setSession($request->getSession());
+
+        return $this->app->handle($subRequest, HttpKernelInterface::SUB_REQUEST, false);
+    }
```

Comment:
This is how [Silex documents it](https://silex.sensiolabs.org/doc/2.0/cookbook/sub_requests.html), and as per the original thread on this bit, the events are still desired … So?